doc/user-guides/acrn-shell.rst

```diff
@@ -21,9 +21,12 @@ The ACRN hypervisor shell supports the following commands:
      - List all vCPUs in all VMs
    * - vcpu_dumpreg <vm_id> <vcpu_id>
      - Dump registers for a specific vCPU
-   * - dumpmem <hva> <length>
-     - Dump host memory, starting at a given address, and for a given length
-       (in bytes)
+   * - dump_host_mem <hva> <length>
+     - Dump host memory, starting at a given address(hexadecimal), and for a given length
+       (decimal in bytes)
+   * - dump_guest_mem <vm_id> <gva> <length>
+     - Dump guest memory, vm id(decimal),starting at a given address(hexadecimal), and for a given length
```

Comment:
Dump a User VM (guest) memory region based on the VM ID (``vm_id``, in decimal), the start of the memory region ``gva`` (in hexadecimal) and its length ``length`` (in bytes, decimal number).

Reply:
will update